DirectProgramming/DPC++FPGA/ReferenceDesigns/gunzip/README.md

```diff
@@ -0,0 +1,281 @@
+# GZIP Decompression
```

Comment:
General comment on this file: there was a recent PR for all the FPGA Readmes: https://github.com/oneapi-src/oneAPI-samples/pull/858  Can you make sure these changes are incorporated into this README?

Reply:
I checked out the recent change. Does all of that information really belong in every README? It is generic to all programs, could it not go in a higher level README?